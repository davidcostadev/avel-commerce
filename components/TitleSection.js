import React from 'react';

import styles from '../assets/scss/App.scss';

const TitleSection = ({ title }) => (
  <div className={styles.titleSection}>
    <h1>{title}</h1>
  </div>
);


export default TitleSection;
