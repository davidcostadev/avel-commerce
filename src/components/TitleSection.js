import React from 'react';

import styles from '../assets/scss/App.scss';

const TitleSection = () => (
  <div className={styles.titleSection}>
    <h1>Inform√°tica</h1>
  </div>
);


export default TitleSection;
