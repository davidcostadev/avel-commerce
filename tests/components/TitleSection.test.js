/* eslint no-undef: "off" */
import React from 'react'
import renderer from 'react-test-renderer'
import TitleSection from '../../components/TitleSection'

describe('Component <TitleSection />', () => {
  it('<TitleSection />', () => {
    const tree = renderer.create((
      <TitleSection
        title="Titulo da Section"
      />
    )).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
