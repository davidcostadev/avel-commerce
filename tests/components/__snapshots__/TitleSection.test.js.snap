// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component <TitleSection /> <TitleSection /> 1`] = `
<div
  className={undefined}
>
  <h1>
    Titulo da Section
  </h1>
</div>
`;
